docs/en/ingest-management/elastic-agent/configuration/elastic-agent-standalone-logging.asciidoc

```diff
@@ -29,6 +29,17 @@ agent.logging.files:
 You can specify the following settings in the Logging section of the
 `elastic-agent.yml` config file.
 
+Some outputs will log raw events on errors like indexing errors in the
```

Comment:
You need to document that the content of the event data logs is not shipped to Elasticsearch when agent monitoring is enabled.

Reply:
I added it on https://github.com/elastic/ingest-docs/pull/1053/commits/a7e3b4ef1d517e32ea21229269d2aad592bf9c82